### PR TITLE
refactoring public folder, gruntfile to match latest meanjs folder structure and files

### PR DESCRIPTION
**Pull request by jayprakash1**
I did some testing of existing workflows and it is working. Please let me know if you have feedback for these changes.

### CODE REVIEW

File: app/controllers/articles.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /**
  * Module dependencies.
  */
-var db = require('../../config/sequelize');
+var db = require('../../config/sequelize'),
+    errorHandler = require('./errors.server.controller');
```
Comment (jpotts18) [inline app/controllers/articles.js]: I am strongly against error codes ever being sent back to client side application. It should only speak HTTP. A 400 or a 500 error can tell the client all it every needs to know. I don't mind the idea of catching errors on the server and logging them for diagnostics later, but don't pass the error on to the client.

File: app/views/layout.server.view.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```
Comment (jpotts18) [inline app/views/layout.server.view.html]: I really like these changes by using Jade more efficiently and demonstrating great use of layouts. :+1:

File: app/views/layout.server.view.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	<header data-ng-include="'/modules/core/views/header.client.view.html'" class="navbar navbar-fixed-top navbar-inverse"></header>
+	<section class="content">
+		<section class="container">
+			{% block content %}{% endblock %}
+		</section>
+	</section>
+
+	<!--Embedding The User Object-->
+	<script type="text/javascript">
+		var user = {{ user | json | safe }};
```
Comment (jpotts18) [inline app/views/layout.server.view.html]: Make sure this doesn't have any data that could be used maliciously.

**Comment by chaudhryjunaid** [comment]
@jpotts18 should I close this one?

**Comment by jpotts18** [comment]
It is probably too far out of sync now... Bummer

**Comment by nhhockeyplayer** [comment]
Hi Friend,

I like your stack. Do you have a version in typescript ? It'd be so much nicer if you did.

I wrote a mini hibernate for RAYTHEON back a few jobs... 

If there is anything I could do to contribute to this stack in typescript I am all hands on.

I was a former committer for trails/tynamo.

I am able to help maintain a general stack... typescript + orm = meanjs

Happy to lend a hand for a typescript rev.
Ken

**Pull request closed without merging**